modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -803,7 +803,7 @@
 			<argument index="0" name="from" type="float" />
 			<argument index="1" name="to" type="float" />
 			<description>
-				Random range, any floating point value between [code]from[/code] and [code]to[/code].
+				Returns a random floating point value between [code]from[/code] and [code]to[/code] (inclusive).
```

Comment:
For me it's not fully clear whether "(inclusive)" is about both `from` and `to` or just about `to`. So I'd opt for being more explicit, something like:
`between [code]from[/code] (inclusive) and [code]to[/code] (inclusive).`
or:
`between [code]from[/code] and [code]to[/code] (both ends inclusive).`

It's also not clear how does this method method behave for `from > to`.